Alt-click to select without seeking

FelipeDefensor opened the pull request:
Depends on #303.

Closes #204.

FelipeDefensor committed:
feat: alt-click to select without seeking

azfoo commented:
please rebase onto the latest dev (or #303 since that is already updated).
also, I'm not sure what you expect alt+ ctrl + click to do. I thought it would let you select multiple without ever moving the playback line but it doesn't let you select multiple and it moves the playback line :(

FelipeDefensor commented:
> I'm not sure what you expect alt+ ctrl + click to do. I thought it would let you select multiple without ever moving the playback line but it doesn't let you select multiple and it moves the playback line :(

It should definitely do what you expect, but I forgot to implement that. Nice catch!

FelipeDefensor committed:
test: avoid patch

FelipeDefensor committed:
test: generalize click_timeline_ui for multiple modifiers

FelipeDefensor commented:
@azfoo: fixed it!

FelipeDefensor commented:
Also, let's release 0.5.11 after this one?

azfoo committed:
test: radical parametrisation

currently failing because some clicks are not actually finding the item to click on.

azfoo committed:
test: include score_tl in parametrised tests

azfoo reviewed (changes requested):
Are we expecting there to be a need for alt-unselect as well?

ps. sorry for the failing tests, see commit message.

azfoo committed:
test: fix modifier list

azfoo committed:
fix: create method to click directly on a ui element

azfoo committed:
refactor: remove unnecessary enum

azfoo committed:
fix: flake8 F824

azfoo committed:
version: 0.5.11
